pyeudiw/satosa/backend.py

```diff
@@ -507,3 +512,21 @@ def handle_error(
             },
             status=err_code
         )
+
+    def state_endpoint(self, context):
+        session_id = context.state["SESSION_ID"]
```

Comment:
here we need more exceptions handling

then we need other status like "forbidden" (403) if that session_id is not bound to any session stored in the DB